cmd/monaco/integrationtest/cleanup.go

```diff
@@ -20,62 +20,44 @@ package integrationtest
 
 import (
 	"fmt"
-	"github.com/dynatrace/dynatrace-configuration-as-code/v2/cmd/monaco/runner"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/cmd/monaco/integrationtest/utils/test"
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/timeutils"
-	"github.com/stretchr/testify/assert"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/require"
 	"path/filepath"
-	"strings"
 	"testing"
-
-	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/log"
-	"github.com/spf13/afero"
 )
 
 // CleanupIntegrationTest deletes all configs that are defined in a test manifest. It uses the CLI runner, to call the
 // deletefile.Command to generate a delete file for the test manifest, and delete.GetDeleteCommand to remove configs using
 // the generated file.
-func CleanupIntegrationTest(t *testing.T, fs afero.Fs, manifestPath string, specificEnvironments []string, suffix string) {
-
-	var envArgs []string
-	if len(specificEnvironments) > 0 {
-		envArgs = []string{
-			"--environment",
-			strings.Join(specificEnvironments, ","),
-		}
+func CleanupIntegrationTest(t *testing.T, fs afero.Fs, manifestPath string, environment string, suffix string) {
+	var env string
+	if len(environment) > 0 {
+		env = fmt.Sprintf("--environment %s", environment)
```

Comment:
this results in square brackets around the elements of `environment` passed to the ui, no?

Reply:
i misread it sorry, then it's fine oc